src/main/java/com/gdschongik/gdsc/domain/event/application/EventParticipationService.java

```diff
@@ -52,6 +56,44 @@ public List<EventParticipableMemberDto> searchParticipableMembers(Long eventId,
                 .toList();
     }
 
+    @Transactional(readOnly = true)
+    public AfterPartyAttendanceResponse getAfterPartyAttendances(Long eventId) {
+        Event event = eventRepository.findById(eventId).orElseThrow(() -> new CustomException(EVENT_NOT_FOUND));
+        validateEventEnabledForAfterParty(event);
+
+        List<EventParticipation> eventParticipations = eventParticipationRepository.findAllByEventId(eventId);
```

Comment:
```suggestion
        List<EventParticipation> eventParticipations = eventParticipationRepository.findAllByEvent(event);
```

엔티티를 넘기는 경우 암시적으로 이미 이벤트는 존재한다는 것을 가정하기 떄문에...
조회 로직의 파라미터가 엔티티인 경우 '이미 존재성은 검증했구나' 를 알 수 있다는 이점이 있습니다

반면 ID로 조회하면 아예 없는 ID로 조회를 시도할 수도 있겠죠
엔티티를 받을 수 있는 걸 굳이 ID만 받도록 더 열어줄 필요가 없을 듯 합니다